CHANGELOG.md

```diff
@@ -1,3 +1,51 @@
+# 1.0.10-rc1 (2020-06-23)
+## Edge Agent
+### Features
+* Update codebase to dotnet 3.1. [f87a18a](https://github.com/Azure/iotedge/commit/f87a18a487ea0c05752254aaba04a4f89028120a)
+* Install Trust Bundle in EdgeAgent and EdgeHub. [4f85dcc](https://github.com/Azure/iotedge/commit/4f85dcc7d3fd6d4772d4a9b86ec3ecad651938fd)
+* Metrics use RocksDB [0d2cf76](https://github.com/Azure/iotedge/commit/0d2cf76ea1da561252a1100b7da5b4f0f5f1d471)
+* Metrics Upload to IoTHub [eff5c85](https://github.com/Azure/iotedge/commit/eff5c859e101d14eda85b80513136a2f3a473892)
+* Add "Cmd", "Entrypoint", and "WorkingDir" translations for Kubernetes. [7cbc607](https://github.com/Azure/iotedge/commit/7cbc607ccc483d6e0ab9642be76c8b2d8bc09605)
+* Add Experimental k8s create option feature for pod security context, resources, volumes, nodeSelector, and strategy. [cf2eba9](https://github.com/Azure/iotedge/commit/cf2eba9518a947bb09ebff5dcd6ae42f66d2d045) [23b40e1](https://github.com/Azure/iotedge/commit/23b40e1bf5e9cd04af1246b2214f0759c6446ea9)
+
+### Bug Fixes
+* Reprovision device for all protocols when the connection status change reason is Bad_Credential. [3601a56](https://github.com/Azure/iotedge/commit/3601a566e728f697176398b7f92deb79b60278fe)
+* Vulnerability issues for docker images. [d88fa52](https://github.com/Azure/iotedge/commit/d88fa52d910a71df0ea7b2d38b1e357514027f38) [7873079](https://github.com/Azure/iotedge/commit/7873079c5a3d4e28dcf6c979a1533d6d950fc428)
+* Update SDK to 1.26.0. [5148ee7](https://github.com/Azure/iotedge/commit/5148ee70b793963f74862e047039a412a488ab73)
+
+## Edge Hub
+### Features
+* Update codebase to dotnet 3.1. [f87a18a](https://github.com/Azure/iotedge/commit/f87a18a487ea0c05752254aaba04a4f89028120a)
+* Install Trust Bundle in EdgeAgent and EdgeHub. [4f85dcc](https://github.com/Azure/iotedge/commit/4f85dcc7d3fd6d4772d4a9b86ec3ecad651938fd)
+* Metrics Upload to IoTHub. [eff5c85](https://github.com/Azure/iotedge/commit/eff5c859e101d14eda85b80513136a2f3a473892)
+* Unify TLS protocol parsing. [f319228](https://github.com/Azure/iotedge/commit/f3192289d29be33222dedb93ce7d49ffc532fcd5)
+* Limit route priority to 0-9. [9cf0203](https://github.com/Azure/iotedge/commit/9cf02037ff0f43dc9f8a1e72cde09fd06507e2c8)
+* Add TimeToLive to PriorityQueue. [2662d9c](https://github.com/Azure/iotedge/commit/2662d9cd46f0abbe91e1e0260b37d1f9372609a7)
+* Support module priority in IoT Edge [6fce17b](https://github.com/Azure/iotedge/commit/6fce17bddff05b6d5f805b43330ec7a25a79ba2a)
+
+### Bug Fixes
+* Update codebase to dotnet 3.1. [f87a18a](https://github.com/Azure/iotedge/commit/f87a18a487ea0c05752254aaba04a4f89028120a)
+* Vulnerability issues for docker images. [d88fa52](https://github.com/Azure/iotedge/commit/d88fa52d910a71df0ea7b2d38b1e357514027f38) [7873079](https://github.com/Azure/iotedge/commit/7873079c5a3d4e28dcf6c979a1533d6d950fc428)
+* Update SDK to 1.26.0. [5148ee7](https://github.com/Azure/iotedge/commit/5148ee70b793963f74862e047039a412a488ab73)
```

Comment:
Recommend calling out which SDK

Reply:
The details you're looking for is already in the description of the commit. I'd rather not including all the details here. If you really expected that level of detail we would have all the commits here + its description and that's  just a bit much.

Reply:
I don't mean all the details, but something like we did earlier:
https://github.com/Azure/iotedge/commit/5148ee70b793963f74862e047039a412a488ab73

Reply:
That would also be true for all of the component on Rust, Helm, dotnet3.1 as well. Again, the links are here for a reference.

Reply:
In my opinion, those details should be described at the commit level. The Changelog is just a TLDR.

Reply:
@philipktlin

Reply:
Let’s be consistent as what we did before. Please mention Azure iot c# sdk in release notes. We should follow up the commit comment later on.  Btw Please ping Varun to review. Thanks.